test/unit/Pool/addAsset.js

```diff
@@ -1,17 +1,37 @@
-const { artifacts } = require('hardhat');
+const { artifacts, web3 } = require('hardhat');
 const {
   constants: { ZERO_ADDRESS },
   ether,
   expectRevert,
 } = require('@openzeppelin/test-helpers');
 const { assert } = require('chai');
+const { hex } = require('../utils').helpers;
 const {
   governanceContracts: [governance],
 } = require('../utils').accounts;
+const { BN } = web3.utils;
+
+const PriceFeedOracle = artifacts.require('PriceFeedOracle');
+const ChainlinkAggregatorMock = artifacts.require('ChainlinkAggregatorMock');
 
 const assetAddress = '0xC0FfEec0ffeeC0FfEec0fFEec0FfeEc0fFEe0000';
 
 describe('addAsset', function () {
+  before(async function () {
+    const { pool, dai, stETH, chainlinkDAI, chainlinkSteth } = this;
+
+    const chainlinkNewAsset = await ChainlinkAggregatorMock.new();
+    await chainlinkNewAsset.setLatestAnswer(new BN((1e18).toString()));
+
+    const priceFeedOracle = await PriceFeedOracle.new(
+      [dai.address, stETH.address, assetAddress],
+      [chainlinkDAI.address, chainlinkSteth.address, chainlinkNewAsset.address],
+      [18, 18, 18],
+    );
+
+    await pool.updateAddressParameters(hex('PRC_FEED'), priceFeedOracle.address, { from: governance });
```

Comment:
The operations in this beforeEach should happen inside setup.js, and setup the price feed correctly from the start, not create one in the setup and then replace it before each test.